src/camp_common.h

```diff
@@ -19,7 +19,9 @@
 #ifdef CAMP_USE_SUNDIALS
 #include <cvode/cvode.h>             /* Protoypes for CVODE fcts., consts.  */
 #include <cvode/cvode_direct.h>      /* CVDls interface                     */
-#include <cvode/cvode_impl.h>        /* CVodeMem structure                  */
+#ifdef CAMP_CUSTOM_CVODE
+#  include <cvode/cvode_impl.h>        /* CVodeMem structure                  */
+#endif
```

Comment:
why does this particular header file need to be optionally included?

Reply:
This header file and the `CVodeMem` type are no longer part of newer SUNDIALS API

![image](https://user-images.githubusercontent.com/89685/183463991-eb80ea06-149e-40ba-8dab-2965e1f245e4.png)

Reply:
ah, ok - makes sense.